CSharp/Library/Microsoft.Bot.Connector.Shared/StateAPI/Models/BotData.cs

```diff
@@ -21,9 +21,8 @@ public partial class BotData
         /// <summary>
         /// Initializes a new instance of the BotData class.
         /// </summary>
-        public BotData(string eTag = default(string), object data = default(object))
+        public BotData(string eTag = @"*", object data = default(object))
```

Comment:
@brandonh-msft -- sorry for the confusion. We're in the process of finalizing a new doc set that will be published next week before build -- the [article that you linked to](https://docs.botframework.com/en-us/csharp/builder/sdkreference/stateapi.html) will be replaced by a new article at that time.  I've verified that the new article (about managing state data) does not specify that ETag should be set to asterisk (*) when saving state data.  So, this documentation issue will no longer exist once the new docs are published next week (and the old docs are decommissioned).